src/utils.ts

```diff
@@ -49,7 +49,7 @@ function injectionCSSCodePlugin(cssToInject: string, styleId: string | null): Pl
                 const cssCode = JSON.stringify(cssToInject.trim());
 
                 return `try{var elementStyle = document.createElement('style');${
-                    typeof styleId == 'string' && styleId.length > 0 ? 'elementStyle.id = "${styleId}";' : ''
+                    typeof styleId == 'string' && styleId.length > 0 ? 'elementStyle.id = `${styleId}`;' : ''
```

Comment:
Thank you for your contribution!
Can you change 'elementStyle.id = `${styleId}`;' to `elementStyle.id = ${styleId};` since it's more clear?

As soon as you have done the fix I will publish the new version